test-suite/tools/update-compat/run.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+exit 0
```

Comment:
Likely this script fails when `context_name != default` , so I disabled it until we can get CI working.

Reply:
Indeed, we run the bisect in a different context, so stuff is in `build/coverage` instead of in `buid/default`.

The script seems to be broken by this due to the test suite.